c/authService.c

```diff
@@ -173,16 +188,208 @@ static int serveAuthCheck(HttpService *service, HttpResponse *res) {
     respondWithError(res, HTTP_STATUS_BAD_REQUEST, "Unexpected access level");
     return 0;
   }
-  /* printf("query: user %s, class %s, entity %s, access %d\n", userName, class,
-      entity, access); */
+  // printf("\n\nquery: user %s, class %s, entity %s, access %d accessStr %s\n", userName, class,
+      // entity, access, accessStr);
   privilegedServerName = getConfiguredProperty(service->server,
       HTTP_SERVER_PRIVILEGED_SERVER_PROPERTY);
   rc = zisCheckEntity(privilegedServerName, userName, class, entity, access,
       &reqStatus);
+      printf("\n\nprivileged server name: %s", privilegedServerName);
   respond(res, rc, &reqStatus);
   return 0;
 }
 
+const char* getProfileNameFromRequest(char *url, char *method, int instanceID) {
+  char type[8] = "NULL"; // core || config || service
```

Comment:
why to use `"NULL"` string everywhere, wouldn't an empty string work?

Reply:
I used "NULL" vs "" because I wanted to differentiate between an empty parameter vs no parameter. I'm not sure of a usecase in the field, but I wanted to have a way for a user URL to distinguish between a parameter that's intentionally empty vs N/A